spdx-utils/src/main/kotlin/model/SpdxDocument.kt

```diff
@@ -138,5 +138,11 @@ data class SpdxDocument(
         require(dataLicense.isNotBlank()) { "The data license must not be blank." }
 
         require(documentNamespace.isNotBlank()) { "The document namespace must not be blank." }
+
```

Comment:
Commit message:
- s/spdx/SPDX/
- Please make link [1] point to the exact comment where this was discussed.
- s/file  was/file was/
- Change "with spdx-java-tool[2]. A cli tool that can be used to verify spdx files with SPDX version 2.2" to "with the spdx-java-tool [2], which is a CLI tool that can be used to verify SPDX version 2.2 files." (not the dor at the end of the sentence).

Reply:
Commit message:
- "needs to be with value" sounds clumsy, how about just "needs to be set"?
- You still missing the space in "spdx-java-tool [2]".